main.py

```diff
@@ -178,13 +180,23 @@ def start_cron():
 
 def custom_data(post):
     return post.date_local >= datetime(2020, 3, 15)
+
+def zip_non_used_files():
+    for hashtag in HASHTAG_LABELS:
+        path = "./#" + hashtag
+        arqZip = zipfile.ZipFile("#" + hashtag + "ZIP.zip", "w", zipfile.ZIP_DEFLATED)
+        for file in os.listdir(path):
+            arqZip.write(os.path.join(path, str(file)), str(file))
+            print("Zipando " + str(file))
+        arqZip.close()
     
 if __name__ == '__main__':
     #download_hashtags_last_7_days(hashtag_labels)
     #os.system('python sort_best.py')
     #'foragarimpoforacovid', 'projetemos', 'memes_forevis' 
-    for hashtag_label in ['projetemos']:
-        download_video(hashtag_label, 10000 ,custom_data)
+    #for hashtag_label in ['projetemos']:
+    #   download_video(hashtag_label, 10000 ,custom_data)
```

Comment:
Why do we commented this piece of code? 🤔

Reply:
This was part of the main, where it was changed just focus on download posts from the hashtag "#projetemos". But the only think the main should do is call "start_cron()", to begin the scheduled activities. I'll make sure to clean this part (and the others with commented code) before the next pull request!